README.md

```diff
@@ -48,6 +48,8 @@ The [tutorial about using VON Network](docs/UsingVONNetwork.md) has information
 
 ## Running the web server in Docker against another ledger
 
+Note that inorder to use Docker Desktop (> version 3.4.0), make sure you uncheck the "Use Docker Compose V2" in Docker Desktop > Preferences > General
+
```

Comment:
Thanks for the contribution.  Sorry it has taken so long to get to the review.

This note would be better located under **VON Network Quick Start Guide** closer to the top of the file.  That way it would be less likely to be missed.

![image](https://user-images.githubusercontent.com/10504350/150817760-c903bc25-367a-4727-87fa-44b26f4269c0.png)

May I also suggest the following edits?
```suggestion
Note that in order to use Docker Desktop (> version 3.4.0), make sure you uncheck the "Use Docker Compose V2" in Docker Desktop > Preferences > General.  Refer to this issue for additional details; [#170](https://github.com/bcgov/von-network/issues/170#issuecomment-972898014)
```